pkg/storkctl/migrationschedule.go

```diff
@@ -196,10 +196,14 @@ func newCreateMigrationScheduleCommand(cmdFactory Factory, ioStreams genericclio
 				var schedulePolicy storkv1.SchedulePolicy
 				schedulePolicy.Name = schedulePolicyName
 				schedulePolicy.Policy = policyItem
-				_, err = storkops.Instance().CreateSchedulePolicy(&schedulePolicy)
-				if err != nil {
-					util.CheckErr(fmt.Errorf("could not create a schedule policy with specified interval: %v", err))
-					return
+				ms, _ := storkops.Instance().GetMigrationSchedule(migrationScheduleName, cmdFactory.GetNamespace())
```

Comment:
I am checking if migrationschedule with given name already exists. 
If it already exists then in the later step when we try to create the migrationschedule it will fail, however in this case according to current code, we would have created a custom schedule policy before this failure which may not be desirable.
To address this issue, I have added a check that if migrationSchedule creation will fail anyways (since migrationschedule already exists) we skip creation of custom interval schedule policy.

Reply:
Have Addressed in the latest commit